eng/install-test-runtimes.ps1

```diff
@@ -11,55 +11,89 @@ param(
 Set-StrictMode -Version Latest
 $ErrorActionPreference="Stop"
 
-$RuntimeVersion11="1.1.13"
+function Get-Latest-Version([string]$channel, [string]$runtime = "", [bool]$coherent = $false) {
+
+    $VersionFileUrl = $null
+    if ($runtime -eq "dotnet") {
+        $VersionFileUrl = "$UncachedFeed/Runtime/$channel/latest.version"
+    }
+    elseif ($runtime -eq "aspnetcore") {
+        $VersionFileUrl = "$UncachedFeed/aspnetcore/Runtime/$channel/latest.version"
+    }
+    elseif ($runtime -eq "") {
+        if ($coherent) {
+            $VersionFileUrl = "$UncachedFeed/Sdk/$Channel/latest.coherent.version"
+        }
+        else {
+            $VersionFileUrl = "$UncachedFeed/Sdk/$Channel/latest.version"
+        }
+    }
+    else {
+        throw "Invalid value for $runtime"
+    }
+
+    $VersionFile = Join-Path -Path $TempDir latest.version
+
+    try {
+        Invoke-WebRequest $VersionFileUrl -OutFile $VersionFile
+    }
+    catch {
+        return ""
+    }
+
+    if (Test-Path $VersionFile) {
+        $VersionText = cat $VersionFile
+        $Data = @($VersionText.Split([char[]]@(), [StringSplitOptions]::RemoveEmptyEntries));
+        return $Data[1].Trim()
+    }
+
+    return ""
+}
+
+$ConfigFile = Join-Path -Path $DotNetDir Debugger.Tests.Versions.txt
+
 $RuntimeVersion21="2.1.12"
-$RuntimeVersion22="2.2.6"
+$AspNetCoreVersion21="2.1.12"
 $RuntimeVersion30="3.0.0"
 $AspNetCoreVersion30="3.0.0"
-$DailyTestText="true"
+$RuntimeVersion31="3.1.0"
```

Comment:
Are we ok not testing live 3.0/3.1 bits? 3.0.1 is already almost out, and just want to know what wee are testing

Reply:
Not sure how to do that since we can get a darc subscription for multiple channels. It will have to be a manual update until we can come up with something better.